addons/website/static/src/common/website_visitor_model.js

```diff
@@ -0,0 +1,16 @@
+import { fields, Record } from "@mail/core/common/record";
+
+export class WebsiteVisitor extends Record {
+    static _name = "website.visitor";
+    static id = "id";
+
+    partner_id = fields.One("Persona");
+    country = fields.One("res.country", {
+        /** @this {import("models").WebsiteVisitor} */
+        compute() {
+            return this.partner_id?.country ?? this.country_id;
+        },
+    });
```

Comment:
You might want to define all the fields that are returned in python so we have proper doc + autocomplete for them (here + d.ts)